server/serverHelpers.js

```diff
@@ -1,38 +1,74 @@
-const getReviews = (product) => {
-  // get all reviews, user info, images, comments for product
-};
+import Promise from 'bluebird';
 
-const getAggregate = (product) => {
-  // get aggregate score & review quantity
-};
+import { db } from '../database/connection';
+
+const getAggregate = product => new Promise((resolve) => {
```

Comment:
Small thing to nitpick at, but maybe set the variable name to productId or something else to make it clear that it's a number.